src/seedu/addressbook/parser/Parser.java

```diff
@@ -137,6 +130,20 @@ private Command prepareAdd(String args) {
         }
     }
 
+    private Command prepareTagged(String args) {
+       // final Matcher matcher = TAGGED_DATA_ARGS_FORMAT.matcher(args.trim());
```

Comment:
Remove unnecessary code completely; you can always recover it from previous versions RCS \#FTW